### PR TITLE
Update NWjs and FFmpeg libraries to 0.72.0

### PR DESCRIPTION
**Pull request by haslinghuis**
Keep in sync with https://github.com/betaflight/betaflight-configurator/pull/3334

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — CHANGES_REQUESTED** [review]
There is a readme with the libraries version in the same folder.

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
